src/components/pages/login.yml

```diff
@@ -1,3 +1,10 @@
 forgotPassword:
   en: Forgot your password?
   de: Kennwort vergessen?
+hintContent:
+  en: |
+    The username is no longer supported for logging in.
+    It continues to be shown as the name under which you author website content.
+  de: |
+    Der Benutzername kann nicht mehr für die Anmeldung benutzt werden.
+    Der bleibt als Autorennamen für alle von Ihnen bearbeiteten Website-Inhalt.
```

Comment:
Autorenname

Reply:
"Er", of course.. I formed the bad habit of saying "Der" in such context all the time.
Thanks, corrected all of the translations you mentioned 🙏